content/en/docs/setup/platform-setup/kubesphere/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: KubeSphere Container Platform
+description: Instructions to setup a KubeSphere Container Platform for Istio.
+weight: 18
```

Comment:
KubeSphere should be after Kubernetes Gardener. Some weights need to be changed for that.